packages/dd-trace/test/appsec/sdk/index.spec.js

```diff
@@ -2,199 +2,220 @@
 
 const proxyquire = require('proxyquire')
 const agent = require('../../plugins/agent')
-const getPort = require('get-port')
 const axios = require('axios')
+const getPort = require('get-port')
 const tracer = require('../../../../../index')
 
-describe('Appsec SDK', () => {
-  describe('calls to external methods', () => {
-    let trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent
-    let appsecSdk
-    const tracer = {}
+describe('Test public API', () => {
+  const tracer = {}
+  const mockReq = { protocol: 'https' }
+  const mockRes = { headersSent: false }
+  const loadTemplates = sinon.stub()
+  let sdk, checkUserAndSetUser, blockRequest, setUser, trackUserLoginSuccessEvent, trackUserLoginFailureEvent,
+    trackCustomEvent
+
+  beforeEach(() => {
+    trackUserLoginSuccessEvent = sinon.stub()
+    trackUserLoginFailureEvent = sinon.stub()
+    trackCustomEvent = sinon.stub()
+    checkUserAndSetUser = sinon.stub()
+    blockRequest = sinon.stub()
+    setUser = sinon.stub()
+
+    const AppsecSdk = proxyquire('../../../src/appsec/sdk', {
+      './track_event': { trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent },
+      './user_blocking': { checkUserAndSetUser, blockRequest },
+      './set_user': { setUser },
+      '../blocking': { loadTemplates }
+    })
 
-    beforeEach(() => {
-      trackUserLoginSuccessEvent = sinon.stub()
-      trackUserLoginFailureEvent = sinon.stub()
-      trackCustomEvent = sinon.stub()
+    sdk = new AppsecSdk(tracer)
+  })
 
-      const AppsecSdk = proxyquire('../../../src/appsec/sdk', {
-        './track_event': { trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent }
-      })
+  it('Test isUserBlocked', () => {
```

Comment:
if you review other test, you could check that this is not the usual format. The test try to do a phrase joining `describe`s and `it`s. 
In this case, it is something like: "Test public API Test isUserBlocked"
I think that we want something like: "test public API isUserBlocked should call checkUserAndSetUser"
The usual format for `it`s is, "methodName should do whatever"